notebooks/00_quick_start/fastai_movielens.ipynb

```diff
@@ -561,7 +519,7 @@
    },
    "outputs": [],
    "source": [
-    "training_removed = pd.concat([users_items, train_valid_df[[USER,ITEM]], train_valid_df[[USER,ITEM]]]).drop_duplicates(keep=False)"
+    "training_removed = pd.concat([users_items, train_valid_df[[USER,ITEM]]]).drop_duplicates(keep=False)"
```

Comment:
Minor point - do we want to use the constants for column names consistently? That is, `reco_utils.common.constants.DEFAULT_USER_COL` for user column, etc.

Reply:
not sure, we are not doing it in the other notebooks. I'm not sure if it is more clear to the user to explicitly provide these values or to leave them with the default values. Not strong preference here tbh

Reply:
My opinion is that we should use these constants everywhere in the "internal" code; but we don't want to use them in notebooks, since the outside users do not know or care about DEFAULT_USER_COL etc. And they will often want to edit the notebooks and replace these names with the ones they have. Otherwise we would parameterize everything in the notebooks and make them look like callable python functions.

Reply:
yeah, that makes sense